sorald-api/src/main/java/sorald/rule/Rule.java

```diff
@@ -13,14 +11,4 @@
 
     /** @return The type of this rule. */
     IRuleType getType();
-
-    /**
-     * Create a rule based on the key.
-     *
-     * @param key A key for which to create a rule.
-     * @return A rule based on the key.
-     */
-    static Rule of(String key) {
-        return new SonarRule(key);
-    }
```

Comment:
It is okay we removed this from the interface. We can instead always use `new SonarRule` in `sorald-sonar`.